tests/foreman/api/test_hostcollection.py

```diff
@@ -66,3 +169,149 @@ def test_positive_read_system_ids(self):
             frozenset((system.id for system in host_collection.system)),
             frozenset((system.id for system in self.systems)),
         )
+
+    @tier1
+    def test_positive_update_name(self):
+        """@Test: Check if host collection name can be updated
+
+        @Feature: Host Collection
+
+        @Assert: Host collection name was successfully updated
+        """
+        host_collection = entities.HostCollection(
+            organization=self.org).create()
+        for new_name in valid_data_list():
+            with self.subTest(new_name):
+                host_collection.name = new_name
+                self.assertEqual(host_collection.update().name, new_name)
+
+    @tier1
+    def test_positive_update_description(self):
+        """@Test: Check if host collection description can be updated
+
+        @Feature: Host Collection
+
+        @Assert: Host collection description was updated
+        """
+        host_collection = entities.HostCollection(
+            organization=self.org).create()
+        for new_desc in valid_data_list():
+            with self.subTest(new_desc):
+                host_collection.description = new_desc
+                self.assertEqual(
+                    host_collection.update().description, new_desc)
+
+    @tier1
+    def test_positive_update_limit(self):
+        """@Test: Check if host collection limit can be updated
+
+        @Feature: Host Collection
+
+        @Assert: Host collection limit was updated
+        """
+        host_collection = entities.HostCollection(
+            organization=self.org,
+            unlimited_content_hosts=False,
+        ).create()
+        for limit in (1, 3, 5, 10, 20):
+            with self.subTest(limit):
+                host_collection.max_content_hosts = limit
+                self.assertEqual(
+                    host_collection.update().max_content_hosts, limit)
+
+    @tier1
+    def test_positive_update_unlimited_content_hosts(self):
```

Comment:
Clarified on IRC, the param name is `unlimited_content_hosts` so the test name like this. Thanks @abalakh